extensions/authentication-jwt/src/keys.ts

```diff
@@ -34,3 +35,64 @@ export namespace UserServiceBindings {
   export const USER_CREDENTIALS_REPOSITORY =
     'repositories.UserCredentialsRepository';
 }
+export namespace RefreshTokenConstants {
+  export const REFRESH_SECRET_VALUE = 'r3fr35htok3n';
+  export const REFRESH_EXPIRES_IN_VALUE = '216000';
+  export const REFRESH_ISSURE_VALUE = 'loopback4';
+}
+
+export namespace RefreshTokenServiceBindings {
+  export const REFRESH_TOKEN_SERVICE = BindingKey.create<RefreshTokenService>(
+    'services.authentication.jwt.refresh.tokenservice',
+  );
+  export const REFRESH_SECRET = BindingKey.create<string>(
+    'authentication.jwt.refresh.secret',
+  );
+  export const REFRESH_EXPIRES_IN = BindingKey.create<string>(
+    'authentication.jwt.referesh.expires.in.seconds',
+  );
+  export const REFRESH_ISSURE = BindingKey.create<string>(
+    'authentication.jwt.referesh.issure',
+  );
+  export const DATASOURCE_NAME = 'refreshdb';
+  export const REFRESH_REPOSITORY = 'repositories.RefreshTokenRepository';
+}
+
+export type RefreshGrant = {
+  refreshToken: string;
+};
+
+export const RefreshGrantSchema = {
+  type: 'object',
+  required: ['refreshToken'],
+  properties: {
+    refreshToken: {
+      type: 'string',
+    },
+  },
+};
+export const RefreshGrantRequestBody = {
+  description: 'Reissuing Acess Token',
+  required: true,
+  content: {
+    'application/json': {schema: RefreshGrantSchema},
+  },
+};
+
+export type TokenObject = {
+  accessToken: string;
+  expiresIn?: string | undefined;
```

Comment:
if it's optional, `undefined` is not needed.
You can use `expiresIn?: string` and `refreshToken?: string`

Reply:
IMO, for feasibility we should keep it is common type that we can use throughout the extension, It also depends, that some user platforms needs expirseIn time to check and refresh the token before it expires.